### PR TITLE
Admin sees all data on admin auctions#show page

### PR DESCRIPTION
**Pull request by jessieay**
* Use same layout as regular admins#show
* Closes https://github.com/18F/micropurchase/issues/773

screen shot 2016-06-28 at 4 20 56 pm

### CODE REVIEW

File: app/presenters/status_presenter/available.rb [hunk 1]
```diff
@@ -11,10 +11,6 @@ def relative_time
     "Time remaining: #{HumanTime.new(time: auction.ended_at).distance_of_time_to_now}"
   end
 
-  def status_text
```
Comment (harrisj) [inline app/presenters/status_presenter/available.rb]: Glad to get rid of these

File: .codeclimate.yml [hunk 2]
```diff
@@ -6,6 +6,7 @@ engines:
     exclude_fingerprints:
     - 508b75e787610c57dc9780baf6be5d74d253df8af35b5c861d6aacc911901a56
     - 8d8bc90f85c52f9aa9cb5be01fb072fc42aa260520c2d8c0adaf6fc8080546cf
+    - 38eaf3a4c38ed941e5397a84c9f95b91a7c3dee9f70c0993131dd3be89180f75
```
Comment (harrisj) [inline .codeclimate.yml]: Is this a CodeClimate issue?

**Comment by harrisj** [comment]
This looks good, but I was wondering if we should get rid of `preview` now?

**Comment by adelevie** [comment]
> This looks good, but I was wondering if we should get rid of preview now?
I *think* so.

**Comment by adelevie** [comment]
🐲

File: .codeclimate.yml (hunk 2 above)
Comment (jessieay) [inline .codeclimate.yml]: yeah we were getting a brakeman error for linking to `issue_url` because it is "user input"

**Comment by jessieay** [comment]
@harrisj @adelevie  the benefit of preview is being able to see non-published auctions as they will look to the vendors. Admin view added in this PR is *similar* to the vendor view, but has a few key differences. Not sure how much preview is being used -- @mtorres253 ?

**Pull request merged**